src/auto-evo/selection_pressure/TemperatureSessilityPressure.cs

```diff
@@ -0,0 +1,71 @@
+ 
+namespace AutoEvo;
+
+using Newtonsoft.Json;
+
+/// <summary>
+///   Selection pressure that favours sessile (non-motile) species in high temperature environments.
+///   This pressure encourages species to become less mobile in hot conditions.
```

Comment:
Doesn't this do the exact *opposite*?

I'll help with getting the style correct now.